_directory/.data.example.yml

```diff
@@ -1,66 +1,26 @@
 ---
-# name of person or company
-name:
+# name (required)
+# may be your real name or your nick
+name: John Doe
+
+# geo position (required)
+# provide your favored level of detail by number of digits (like 50.107811 vs 50.11)
+latitude: 50.11
+longitude: 8.67
+
+# bio
+# describe yourself, your motivation in REDAXO or whatever is relevant for you. should be the size of a tweet (140 chars).
+bio: "Hey, I’m John, englishman from Frankfurt am Main working for YAKAMOTO. I was involved into several REDAXO addons like structure 4.0, the wordpress importer and twitteredaxo. Find me at slack and in the forum."
+
+# image/avatar
+# use file from current folder, should be about 300 x 300 px
+image: johndoe.jpg
+
+# links
+# one or more links to your website, your github profile, twitter, facebook et al.
+links:
```

Comment:
macht es sinn ggf. eine gravatar url anzugeben und das logo davon dann in der karte zu sehen?

Reply:
Gute Idee! Am besten sollte beides möglich sein, und wir könnten es vermutlich mit einer Value abhandeln.

Reply:
Sorry, das kam jetzt nicht richtig rüber. Ich wollte sagen: Wir können den Gravatar mit am Bild abfeiern, er braucht nicht unten in die Link-Section.

Reply:
und würde man dann das gravatar bild auf der karte sehen wenn man nah genug rannzoomed? (würde ein wenig die individualität betonen)

Reply:
Gute Idee. Weiß nicht, wie gut und einfach das mit Leaflet geht, und ob es nicht zuviel Traffic produziert, wenn z. B. 300 Thumbnails geladen werden müssen beim Aufruf der Karte. Aber sollten wir als Feature prüfen!